hibernate-reactive-core/src/main/java/org/hibernate/reactive/bulk/impl/ReactiveIdTableSupport.java

```diff
@@ -28,7 +29,7 @@ public ReactiveIdTableSupport(Dialect dialect) {
 
     @Override
     public String generateIdTableName(String baseName) {
-        return "ht_" + baseName;
+        return (dialect instanceof SQLServerDialect ?  "#" : "ht_") + baseName;
```

Comment:
So ... we're not using SQL Server temporary tables anymore? We're using a regular table?

Reply:
Ooops, sorry, my mistake, I misread the diff.